ui/src/components/forms/SegmentsPicker.tsx

```diff
@@ -28,7 +26,11 @@ export default function SegmentsPicker({
     new Set<string>(parentSegments.map((s) => s.key))
   );
 
-  const [editing, setEditing] = useState<boolean>(editMode);
+  const [editing, setEditing] = useState<boolean>(true);
+
+  useEffect(() => {
+    setEditing(true);
+  }, [parentSegments]);
```

Comment:
i do still wonder if the same thing can be accomplished by using `useState` for the segments storage, but if it works then it works

Reply:
@markphelps It probably can 🤔  I wanted to keep the state in one place to avoid all sorts of weird things happening. But it probably can be done like that.

Also, I wanted to be able to tie the state of the selected segments to the formik property.